bfxapi/models/withdraw.py

```diff
@@ -42,7 +42,8 @@ def from_raw_withdraw(raw_withdraw):
         method = raw_withdraw[WithdrawModel.METHOD]
         wallet = raw_withdraw[WithdrawModel.WALLET]
         amount = raw_withdraw[WithdrawModel.AMOUNT]
-        return Withdraw(w_id, method, wallet, amount)
+        fee = raw_withdraw[WithdrawModel.FEE]
```

Comment:
@itsdeka if this is the withdraw method fee seems to be 8th item in array:
```
  [
    WITHDRAWAL_ID,
    _PLACEHOLDER
    METHOD,
    PAYMENT_ID,
    WALLET,
    AMOUNT,
    _PLACEHOLDER,
    _PLACEHOLDER,
    WITHDRAWAL_FEE
  ]
```
see full doc https://docs.bitfinex.com/reference/rest-auth-withdraw

Reply:
You are right

Withdrawal = generate_labeler_serializer("Withdrawal", klass=types.Withdrawal, labels=[
    "withdrawal_id",
    "_PLACEHOLDER",
    "method",
    "payment_id",
    "wallet",
    "amount",
    "_PLACEHOLDER",
    "_PLACEHOLDER",
    "withdrawal_fee"
])

In the new lib is correct, idk how I missed it here